packages/tools/src/index.ts

```diff
@@ -3,5 +3,8 @@ export {default as groupFilesByType} from './groupFilesByType';
 export {default as isPackagerRunning} from './isPackagerRunning';
 export {default as getDefaultUserTerminal} from './getDefaultUserTerminal';
 export {fetch, fetchToTemp} from './fetch';
+export {default as launchDefaultBrowser} from './launchDefaultBrowser';
+export {default as launchDebugger} from './launchDebugger';
+export {default as launchEditor} from './launchEditor';
```

Comment:
Were these moved here to be reused by Expo?

Reply:
These were moved from `cli` to avoid circular dependency because `dev-server-api` uses them.